GVFS/GVFS.Notifications/VFSForGit.Mac/VFSForGit.UnitTests/Mocks/MockAboutWindowController.m

```diff
@@ -0,0 +1,23 @@
+#import <Foundation/Foundation.h>
+#import "MockAboutWindowController.h"
+
+@interface MockAboutWindowController()
+
+@property (readwrite) BOOL aboutBoxDisplayed;
+
+@end
+
+@implementation MockAboutWindowController
```

Comment:
FYI, consider using OCMock, it can reduce or eliminate the need for implementing your own mock classes.

Reply:
Agree. I did evaluate `OCMock` option. Since this is a tiny project, I decided to use the pattern we follow in other (like `ProjFS`) Mac projects - to create our own mocks.